src/compiler/program.ts

```diff
@@ -3156,6 +3156,10 @@ namespace ts {
                     createDiagnosticForOptionName(Diagnostics.Option_isolatedModules_can_only_be_used_when_either_option_module_is_provided_or_option_target_is_ES2015_or_higher, "isolatedModules", "target");
                 }
 
+                if (options.preserveConstEnums === false) {
+                    createDiagnosticForOptionName(Diagnostics.Option_preserveConstEnums_cannot_be_disabled_when_isolatedModules_is_enabled, "preserveConstEnums", "isolatedModules");

```

Comment:
Let’s get a test that shows this, and then we should be good to go. Thanks!

Reply:
I added a "compiler" test, because I couldn't find any other ways to test command line arguments, and/or I couldn't get any of the other test frameworks to work.